### ISSUE TITLE
Updated on Alexa2 3.17.4: Placing a command in e.g. "textCommand" does no longer play this command

### ISSUE THREAD

**Issue by MichaelSchaaf999** [Apollon77/ioBroker.alexa2#921]
**Check if the Bug maybe is with Amazon**  
The speak command of the same or similar content direct verbally into the Echo device works fine.

**Describe the bug**  
Entering the command in text mode in the "textCommand" field and executing this does not work.

**To Reproduce**  

**Expected behavior**  

**Screenshots & Logfiles**  

**Versions:**  
 - Adapter version: 3.17.4
 - JS-Controller version: v14.19.0
 - Node version: 4.0.23
 - Operating system: macOS Monterey 12.4

**Additional context**

**Comment by Apollon77** [comment]
So textcommand in general works, just some commands not? Then it is an Amazon topic where I can not do anything ... With the current issues because of the bug (see pinned isse) better wait until the situation has normalized ... Else no idea ...  You can check the debug log if an error was returned by Amazon when sending the command

**Comment by MichaelSchaaf999** [comment]
Thanks, @Apollon77 :  Waiting for Amazon for sure is an option.
Question: Which debug log are you talking about? iOBroker log? Is there an Amazon Echo log? If so: Where?

**Comment by MichaelSchaaf999** [comment]
Thx. Done. Literally millions of messages occur.

**Comment by Apollon77** [comment]
Yes. especially the adapter startup is very heavy ... simpyl wait untill that is all done then it shozld get more calm in the log and you can try a textCommmand

**Comment by MichaelSchaaf999** [comment]
Nochmal ich hier, sorry.

Im LiesMich zu dem Adapter steht dies:
<img width="750" alt="image" src="https://user-images.githubusercontent.com/53481090/184383664-74198ec5-5c37-4c16-9ae3-a061bd608823.png">
Aber das ist nicht mehr ganz aktuell, oder?

Der textCommand "Display aus" funktioniert nämlich einwandfrei.

Der textCommand "zeige Startbildschirm" (um den Bildschirm dann bspw morgens wieder einzuschalten) funktioniert leider nicht:

_State changed alexa2.0.History.json: {"val":"{\"name\":\"WoZi Echo Show 8 2nd Gen\",\"serialNumber\":\"G091GM07123303XX\",\"summary\":\"zeige startbildschirm\",\"creationTime\":1660316207031,\"domainApplicationId\":\"\",\"domainApplicationName\":\"\",\"cardContent\":\"\",\"card\":\"\",\"answerText\":\"Da bin ich mir leider nicht sicher.\",\"utteranceType\":\"ROUTINES_OR_TAP_TO_ALEXA\",\"domain\":\"Knowledge\",\"intent\":\"QAIntent\"}","ack":true,"ts":1660316208636,"q":0,"from":"system.adapter.alexa2.0","user":"system.user.admin","lc":1660316208636}_

Ich vermute mal stark, dass das kein Adapter-Issue ist: Aber was kann man tun, via textCommand, um den Bildschirm wieder einzuschalten? Wenn ich in das Device "Echo" reinbrülle, dann geht der Bildschirm ordnungsgemäß an.
Wenn ich dies Schlüsselwort "Echo" in das Objekt textCommand einstelle, dann kommt die o.a. Meldung "Da bin ich mir leider nicht sicher".

Mache ich etwas falsch?

LG
Michael

**Comment by Apollon77** [comment]
@MichaelSchaaf999 Was bitte hat diese Frage mit dem Issue thema zu tun? Bitte eigenes Issue anlegen
